### PR TITLE
Monodevelop + Mono upgrade

### PR DESCRIPTION
**Pull request by obadz**

### CODE REVIEW

**Comment by lethalman** [comment]
I believe this PR has other changes that are not related to the mono updates.

**Comment by obadz** [comment]
Yes sorry but they were already merged so shouldn't cause any diff?

**Comment by lethalman** [comment]
It's hard to review the PR with those changes, would you rebase please?

**Comment by obadz** [comment]
Ok will do

**Comment by obadz** [comment]
Rebased.

File: pkgs/development/libraries/gnome-sharp/default.nix [hunk 1]
```diff
@@ -12,6 +12,8 @@ stdenv.mkDerivation {
 
   patches = [ ./Makefile.in.patch ];
 
+  dontStrip = true;
```
Comment (lethalman) [inline pkgs/development/libraries/gnome-sharp/default.nix]: Why?

Comment (obadz) [inline pkgs/development/libraries/gnome-sharp/default.nix]: Without it, I get there same error as: http://stackoverflow.com/questions/1428535/mono-missing-file-gnome-sharp-at-the-momment-to-compile-the-gnome-sample-appli (as if gnome-sharp wasn't there) when building monodevelop.

**Comment by lethalman** [comment]
Built it locally, seems fine to me. Thanks.

**Pull request merged**